### ISSUE TITLE
db.valueSize(key = 1) paramer is value

### ISSUE THREAD

**Issue by javadev** [simerplaha/SwayDB.io#8]
http://www.swaydb.io/api/read/valueSize/

**Comment by simerplaha** [comment]
Yes that's correct. It returns the byte size of the input value.

**Comment by javadev** [comment]
In documentation parameter is key but in code parameter is value

**Comment by simerplaha** [comment]
Good catch! Yea it should say `value`. Will fix the documentation.
